addons/point_of_sale/static/src/app/screens/product_screen/product_info_popup/product_info_popup.xml

```diff
@@ -2,8 +2,8 @@
 <templates id="template" xml:space="preserve">
     <t t-name="point_of_sale.ProductInfoPopup">
         <Dialog title.translate="Product information">
-            <ProductInfoBanner t-if="!props.product.isConfigurable()" product="props.product" info="props.info"/>
-            <div class="section-inventory mt-3 mb-4 pb-4 border-bottom text-start" t-if="!isVariant and props.info.productInfo.warehouses.length > 0">
+            <ProductInfoBanner t-if="!props.productTemplate.isConfigurable()" productTemplate="props.productTemplate" product="props.productTemplate.product_variant_ids[0]" info="props.info"/>
```

Comment:
```suggestion
            <ProductInfoBanner t-if="!props.productTemplate.isConfigurable()" productTemplate="props.productTemplate" info="props.info"/>
```

maybe you don't need to specify the `product` since it's optional. Besides, we like product template information here.